WooCommerce/Classes/ViewRelated/Orders/Simple Payments/SimplePaymentsSummary.swift

```diff
@@ -0,0 +1,256 @@
+import SwiftUI
+
+/// View to summarize the Simple Payments order to be created
+///
+struct SimplePaymentsSummary: View {
+
+    /// Order note content
+    ///
+    let noteContent: String?
+
+    var body: some View {
+        VStack {
+            ScrollView {
+                VStack(spacing: Layout.noSpacing) {
+
+                    CustomAmountSection()
+
+                    Spacer(minLength: Layout.spacerHeight)
+
+                    EmailSection()
+
+                    Spacer(minLength: Layout.spacerHeight)
+
+                    PaymentsSection()
+
+                    Spacer(minLength: Layout.spacerHeight)
+
+                    NoteSection(content: noteContent)
+                }
+            }
+
+            TakePaymentSection()
+        }
+        .background(Color(.listBackground))
+    }
+}
+
+/// Represents the Custom amount section
+///
+private struct CustomAmountSection: View {
+    var body: some View {
+        Group {
+            Divider()
+
+            AdaptiveStack(horizontalAlignment: .leading, spacing: SimplePaymentsSummary.Layout.horizontalStackSpacing) {
+                Image(uiImage: .priceImage)
+                    .padding()
+                    .foregroundColor(Color(.systemGray))
+                    .background(Color(.listBackground))
+
+                Text(SimplePaymentsSummary.Localization.customAmount)
+                    .headlineStyle()
+
+                // Temporary data
+                Text("$40.00")
+                    .headlineStyle()
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            .bodyStyle()
+            .padding()
+            .background(Color(.listForeground))
+
+            Divider()
+        }
+    }
+}
+
+/// Represents the email section
+///
+private struct EmailSection: View {
+    var body: some View {
+        Group {
+            Divider()
+
+            TitleAndTextFieldRow(title: SimplePaymentsSummary.Localization.email,
+                                 placeholder: SimplePaymentsSummary.Localization.emailPlaceHolder,
+                                 text: .constant("")) // Temporary data
+                .background(Color(.listForeground))
+
+            Divider()
+        }
+    }
+}
+
+/// Represents the Payments Section
+///
+private struct PaymentsSection: View {
+    var body: some View {
+        Group {
+            Divider()
+
+            VStack(alignment: .leading, spacing: SimplePaymentsSummary.Layout.verticalSummarySpacing) {
+
+                Text(SimplePaymentsSummary.Localization.payment)
+                    .headlineStyle()
+                    .padding([.horizontal, .top])
+
+                // Temporary data
+                TitleAndValueRow(title: SimplePaymentsSummary.Localization.subtotal, value: .content("$40.0"), selectable: false) {}
```

Comment:
In a live preview, I noticed that even though the `TitleAndValueRow` is set to `selectable: false`, the button is still tappable (notice that "Subtotal" and "Order Total" respond to taps):

<img src="https://user-images.githubusercontent.com/8658164/140937813-f487ad35-f118-412c-aacd-4b74c851e651.gif" width="300px">

I think probably we should update the `TitleAndValueRow` component to set `disabled(!selectable)` on the button, so those rows don't respond to taps when they aren't selectable.

Reply:
Another nice catch @rachelmcr! Fixed on https://github.com/woocommerce/woocommerce-ios/pull/5377/commits/f1ac8e54f472abf954baea8597679d5c3d63f4fd

https://user-images.githubusercontent.com/562080/140952084-381bd981-0205-4451-88c7-e09a0dfee9bb.mov